### PR TITLE
update class names to highlight

### PR DESCRIPTION
**Pull request by gvonkoss**

### CODE REVIEW

File: src/scss/component/_syntax-highlight.scss [hunk 1]
```diff
@@ -19,18 +19,26 @@
 	overflow-x: auto;
 }
 
-.hljs-attribute {
+.hljs-string {
+  color: #c54700; //Bump color for wcag AA
```
Comment (notlee) [inline src/scss/component/_syntax-highlight.scss]: What is this colour based on?

Comment (gvonkoss) [reply src/scss/component/_syntax-highlight.scss]: uh... copy pasta. 🙈

Comment (notlee) [reply src/scss/component/_syntax-highlight.scss]: ... out of interest. mandarin?

Comment (gvonkoss) [reply src/scss/component/_syntax-highlight.scss]: 🦅 👀

**Pull request merged**